GeometryCore/include/Triangle.h

```diff
@@ -15,10 +14,14 @@ namespace geom_utils
 
     struct Triangle3D
     {
+        Triangle3D() {};
         Triangle3D(const FPoint3D& a, const FPoint3D& b, const FPoint3D& c)
             : a(a), b(b), c(c)
         {}
 
         FPoint3D a, b, c;
     };
+
+    FPoint3D getNormal(const Triangle3D& triangel);
+    FPoint2D getNormal(const Triangle2D& triangel);
```

Comment:
I'm thinking... that normal for 2D triangle in 2D has no sense.. So remove this

Reply:
done